package.json

```diff
@@ -56,7 +56,7 @@
     "eslint-import-resolver-typescript": "2.0.0",
     "eslint-plugin-filenames": "1.3.2",
     "eslint-plugin-import": "2.19.1",
-    "eslint-plugin-jsdoc": "35.1.2",
+    "eslint-plugin-jsdoc": "35.1.3",
```

Comment:
@rvl I think you forgot to push this dependency (the `tar` file) and this will fail when running `yarn --offline`. The reason behind pushing this dependencies is that it will help generating reproducible builds and it works quite well with `nix` 

https://classic.yarnpkg.com/blog/2016/11/24/offline-mirror/

Reply:
btw, I have a fix for this. @rvl just let me know and I can push it here.

Reply:
Please do - I didn't know about the offline caching.

Reply:
Was this package bumped for a reason? I'm guessing it was removed and re-added which would have pulled the new version